monstereo/visuals/printer.py

```diff
@@ -165,6 +239,83 @@ def factory_axes(self, dic_out):
             axes.append(ax1)
         return figures, axes
 
+
+    def draw_social(self, figures, axes, image, dic_out, annotations):
+
+        angles = dic_out['angles']
+        sizes = [abs(dic_out['uv_heads'][idx][1] - uv_s[1]) / 1.5 for idx, uv_s in
+                 enumerate(dic_out['uv_shoulders'])]
+
```

Comment:
Same thing, if you can please make this separate from social distancing. It could also be called draw_activities and then you have the different options (both social distancing and raise hand or only raise hand etc)